python/dask_cudf/dask_cudf/sorting.py

```diff
@@ -62,6 +62,7 @@ def _simple_shuffle(df, columns, npartitions, ignore_index=True):
 
     token = tokenize(df, columns)
     simple_shuffle_group_token = "simple-shuffle-group-" + token
+    df_nparitions = set(range(df.npartitions))
```

Comment:
A set feels like it would be slower than an array-like of a list or tuple, is this not the case?